src/main/java/marquez/api/resources/DatasourceResource.java

```diff
@@ -28,6 +34,9 @@ public DatasourceResource() {}
   public Response list(
       @QueryParam("limit") @DefaultValue("100") Integer limit,
       @QueryParam("offset") @DefaultValue("0") Integer offset) {
-    return Response.ok(new DatasourcesResponse(Collections.emptyList())).build();
+
+    List<marquez.service.models.Datasource> datasourceList = datasourceService.list(limit, offset);
```

Comment:
Is there a reason we are specifying the fully-qualified path for `Datasource`?

Reply:
minor: we know that `datasourceList` is a `List`, therefore the suffix is redundant, meaning we can just take the plural form: `datasources`

Reply:
Good catch. No reason for that to be fully-qualified. Will also rename to datasources. This'll be fixed in the next revision.

Reply:
Please see PRs #394, #387, and #384, where this is fixed.